docs/sessions.txt

```diff
@@ -69,10 +69,10 @@ following ways:
 The following list describes other driver options that you can set in
 the ``options`` array:
 
-- ``id_field``: Field name for storing the session ID (default: ``_id``)
-- ``data_field``: Field name for storing the session data (default: ``data``)
-- ``time_field``: Field name for storing the timestamp (default: ``time``)
-- ``expiry_field``: Field name for storing the expiry-timestamp (default: ``expires_at``)
+- ``id_field``: Custom field name for storing the session ID (default: ``_id``)
```

Comment:
Fixes to #3254 that JT suggested

Reply:
Thanks, but this is not related to this PR. It should be in a dedicated PR.

Reply:
sure no problem